slcp/cli.py

```diff
@@ -60,7 +60,16 @@ def __init__(self):
             help="process only files without given extensions",
         )
         self.parser.add_argument(
-            "-m", "--move", action="store_true", help="move files instead of copying"
+            "-pm",
+            "--preservemetadata",
+            action="store_true",
+            help="try to preserve files and folders metadata",
+        )
+        self.parser.add_argument(
+            "-m",
+            "--move",
+            action="store_true",
+            help="move files instead of copying"
         )
```

Comment:
It think it would be better to create a mutually exclusive group for `-m` and `-pm` options. Like it is done for `-sc` and `-dc` here: https://github.com/pltnk/selective_copy/blob/18f6786d04afe5aacad0004e942ba2194392e4d8/slcp/cli.py#L37

What do you think?

Reply:
You're right, I agree.